packages/scripts/src/dbt/generateModel.js

```diff
@@ -79,6 +81,24 @@ async function getColumnsInRelation(client, schemaName, tableName) {
   }));
 }
 
+/**
+ * @param {pg.Client} client
+ * @param {number} oid
+ * @returns {Array<{ oid: number; name: string; }>} A list of triggers that exist on the given table.
```

Comment:
Very much a nitpick, but do we need to be returning the `oid` here? I found the code a little tricky to follow but it looks like we never use it, and always end up calling `triggers.map(t => t.name)` at some point. Wondering if we could just return `Array<string>` of trigger names here?

Reply:
Do understand if we just want to keep it consistent with the other functions though :+1: